index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Todo App</title>
+    <script src="app.js" defer></script>
+  </head>
+  <body>
+    <form onsubmit="addItemHandler(event)">
+      <input type="text" id="userInput" />
+      <button id="FormSubmit">Submit</button>
+    </form>
+    <ul id="list"><ff/ul>
```

Comment:
@Anzormumladze wtf is `<ff/ul>`?
<img src="http://i.imgur.com/JGTj2xZ.png" />

Reply:
:D pakom gadairbina